cli/src/main/java/com/devonfw/tools/ide/cli/Ideasy.java

```diff
@@ -19,6 +22,8 @@ public final class Ideasy {
 
   private AbstractIdeContext context;
 
+  private static final Logger logger = LoggerFactory.getLogger(Ideasy.class);
```

Comment:
I do not see that this is used anywhere. What is the value then?
Leftover from local dev test?

Reply:
Jup, removed it now.